hack/verify-bazel.sh

```diff
@@ -17,7 +17,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_ROOT
```

Comment:
Combine with previous line

Reply:
@fejta https://github.com/koalaman/shellcheck/wiki/SC2155#declare-and-assign-separately-to-avoid-masking-return-values ?
`export` always returns true, so having it on one line swallows the exit code, defeating errexit.